gradle/config/pmd/pmd.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="Base ruleset" xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"

```

Comment:
Nice! I tried and utterly failed to find a good PMD template for this config file in the past. May steal this.